deployment/templates/daemonset.yaml

```diff
@@ -42,6 +42,9 @@ spec:
         {{- toYaml .Values.podAnnotations | nindent 8 }}
       {{- end }}
     spec:
+      {{- with .Values.runtimeClassName }}
```

Comment:
Could you please modify this so that the `runtimeClassName` is not included if the `.Values.runtimeClassName` is blank? This aligns with the behavior of the code surrounding it. Thank you.
```
{{ - if .Values.runtimeClassName }}
runtimeClassName: {{ .Values.runtimeClassName }}
{{- end }}
```